readme.md

```diff
@@ -155,6 +168,10 @@ Size of the queue.
 
 Number of pending promises.
 
+#### .timeout
+
+Get most recent timeout time in millis or set the timeout for future operations.
```

Comment:
You can just link to the `timeout` option. No description needed.

```
#### [.timeout](#timeout)
```

In the future we could expose all options that can be modified through `.options`. Or you can do it now if you want to :)

Reply:
Good idea to open for options, which ones would that be though?

All of `concurrency`, `intervalCap`, `interval`, `carryoverConcurrencyCount`, `throwOnTimeout`?

Reply:
Yup, except `throwOnTimeout`.